basic-features.md

```diff
@@ -4,77 +4,125 @@ summary: Learn about the basic features of TiDB.
+| [`ascii`](/character-set-and-collation.md) [^1]                                      | Y     | Y   | Y   | Y   | Y   |
+| [`latin1`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| [`binary`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| **Collations**                                                                       | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [`utf8_bin`](/character-set-and-collation.md)                                        | Y     | Y   | Y   | Y   | Y   |
+| [`utf8_general_ci`](/character-set-and-collation.md)                                 | Experimental | Experimental | N   | N   | N   |
+| [`utf8_unicode_ci`](/character-set-and-collation.md)                                 | Experimental | Experimental | N   | N   | N   |
+| [`utf8mb4_bin`](/character-set-and-collation.md)                                     | Y     | Y   | Y   | Y   | Y   |
+| [`utf8mb4_general_ci`](/character-set-and-collation.md)                              | Experimental     | Experimental   | N   | N   | N  |
+| [`utf8mb4_unicode_ci`](/character-set-and-collation.md)                              | Experimental     | Experimental   | N   | N   | N  |
+| [`ascii_bin`](/character-set-and-collation.md)                                       | Y     | Y   | Y   | Y   | Y   |
+| [`latin1_bin`](/character-set-and-collation.md)                                      | Y     | Y   | Y   | Y   | Y   |
+| [`binary`](/character-set-and-collation.md)                                          | Y     | Y   | Y   | Y   | Y   |
+| **Indexing and constraints**                                                         | **5.0** | **4.0** | **3.1** | **3.0** | **2.1** |
+| [Expression Indexes](/sql-statements/sql-statement-create-index.md#expression-index) | Experimental | Experimental | N   | N   | N   |
+| [Columnar Storage (TiFlash)](/tiflash/tiflash-overview.md)                           | Y     | Y   | Y   | N   | N   |
+| [RocksDB Engine](/storage-engine/rocksdb-overview.md)                                | Y     | Y   | Y   | Y   | Y   |
+| [Titan Plugin](/storage-engine/titan-overview.md)                                    | Y     | Y   | Experimental | Experimental | Experimental |
+| [Invisible Indexes](/sql-statements/sql-statement-add-index.md)                      | Y     | N   | N   | N   | N   |
+| [Composite `PRIMARY KEY`](/constraints.md)                                           | Y     | Y   | Y   | Y   | Y   |
+| [Unique indexes](/constraints.md)                                                    | Y     | Y   | Y   | Y   | Y   |
+| [Clustered index on integer `PRIMARY KEY`](/constraints.md)                          | Y     | Y   | Y   | Y   | Y   |
+| [Clustered index on composite or non-integer key](/constraints.md)                   | Experimental | N | N | N | N  |
```

Comment:
IIUC, we are going to GA this feature in TiDB 5.0. @zhangjinpeng1987 please confirm.

Reply:
Confirmed with @zhangjinpeng1987, yes.

Reply:
for `Clustered index on composite or non-integer key`, how about mark it as `N` at present since we haven't supported this feature yet.